com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py

```diff
@@ -24,31 +21,39 @@ class StreamsConnection:
     information.
 
     Example:
+        >>> # If the resource url does not have a valid SSL certificate, you can disable the SSL verification warning
+        >>> from requests.packages import urllib3
+        >>> urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        >>>
         >>> _resource_url = "https://streamsqse.localdomain:8443/streams/rest/resources"
-        >>> sc = StreamsConnection(username="streamsadmin", password="passw0rd", resource_url=_resource_url)
+        >>> # By default SSL verification is enabled, set verify=False to disable it
+        >>> sc = StreamsConnection(username="streamsadmin", password="passw0rd", resource_url=_resource_url, verify=False)
         >>> instances = sc.get_instances()
         >>> jobs_count = 0
         >>> for instance in instances:
         ...    jobs_count += len(instance.get_jobs())
         >>> print("There are " + jobs_count + " jobs across all instances.")
-
     """
-    def __init__(self, username=None, password=None, resource_url=None, config=None, instance_id=None):
+    def __init__(self, username=None, password=None, resource_url=None, config=None, instance_id=None, **session_kwargs):
```

Comment:
`session_kwargs` contains the list of keyword arguments that can be supplied to the Requests session object that `StreamsConnection` creates.